src/librustdoc/passes/calculate_doc_coverage.rs

```diff
@@ -18,6 +18,18 @@ pub const CALCULATE_DOC_COVERAGE: Pass = Pass {
     description: "counts the number of items with and without documentation",
 };
 
+struct Tests {
+    found_tests: usize,
+}
+
+impl crate::doctest::Tester for Tests {
+    fn add_test(&mut self, _: String, config: LangString, _: usize) {
+        if config.rust {
+            self.found_tests += 1;
```

Comment:
Why is this different for doc coverage than for the lint? Shouldn't they be the same?

Reply:
Not *exactly* in my opinion. A private doc test should complain when a runnable test is present whereas the doc coverage should just care if it's a rust codeblock or not (because even if ignored, it remains a code example!). What do you think?

Reply:
I think they should be the same. `ignore` doesn't mean that it's a code example, just that it should be highlighted like rust. Consider for example https://yottadb.gitlab.io/Lang/YDBRust/yottadb/macro.ci_t.html: it uses `ignore`, but it is not valid rust code and won't compile. So it shouldn't be reported as a doc test.

Reply:
Fine by me! Back to one code then. ;)